robot/web/stream/state/state.go

```diff
@@ -185,6 +200,10 @@ func (state *StreamState) sourceEventHandler() {
 			if state.activeClients == 0 {
 				state.tick()
 			}
+		case msgTypeResize:
+			state.logger.Debug("resize event received")
+			state.isResized = true
```

Comment:
Can we ever go back to not resized?

Reply:
Thanks for bringing this up -- the Reset path has been implemented to handle getting out of resized mode.